addons/project/models/project.py

```diff
@@ -925,18 +925,6 @@ def _get_stat_buttons(self):
                 'show': True,
                 'sequence': 60,
             })
-            buttons.append({
```

Comment:
The commit title should start by a verb like stated in the [guidelines](https://www.odoo.com/documentation/16.0/contributing/development/coding_guidelines.html#commit-message-header)
This is just section in a more generic guideline on commit. It worth reading it completely.

Don't hesitate to share those to your colleagues as commit messages are a recurrent issue.

Thanks! 🙂

Reply:
@IT-Ideas , Done all the changes. Thanks